tests/expr_and_series/over_test.py

```diff
@@ -68,3 +68,147 @@ def test_over_invalid(request: pytest.FixtureRequest, constructor: Constructor)
     df = nw.from_native(constructor(data))
     with pytest.raises(ValueError, match="Anonymous expressions"):
         df.with_columns(c_min=nw.all().min().over("a", "b"))
+
```

Comment:
Generic feedback for the tests, arrow will/is failing, you may need to add it to the check for which context to use

Reply:
Thanks a lot. I have updated the context accordingly.